lib/utils/fields.go

```diff
@@ -422,6 +434,9 @@ func containsAll(slice []string, items []string) bool {
 
 func containsAny(slice []string, items []string) bool {
 	set := setutils.New(slice...)
+	if len(items) == 0 {
+		return false
```

Comment:
...whereas here, returning `false` is completely valid, as instead of using `and`, we would use `or` for the definition of the operation, and here, the neutral element for the logical sum is `false`. However, this will also mean that this conditional statement is unnecessary, as the loop below will simply have zero iterations.